basescript/basescript.py

```diff
@@ -182,3 +193,10 @@ def run(self):
         Override this method to define logic for `run` sub-command
         """
         pass
+
+    def pretty(self):
+        print("hello")
```

Comment:
Is this a dummy definition? Then I think we should use `pass` instead of a print statement

Reply:
Thanks for catching that. I had initially thought of the writing the sub-command implementation in that function but later decided to do inside `pretty_print` from log module. Removed it now in the latest commit.